.eslintrc.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
                 'jQuery': 'readonly',
                 // Jellyfin globals
                 'ApiClient': 'writable',
+                'events': 'writable',
```

Comment:
I think using `Events` might be more consistent with the other module names that are exported?

Reply:
https://github.com/jellyfin/jellyfin-web/pull/2707/commits/d6c00aa6f40efbf41e0acd6c6398b6f356b787e5